corehq/apps/app_manager/tests/templatetags/test_xforms_extras.py

```diff
@@ -0,0 +1,372 @@
+from lxml.html import fragment_fromstring
+from django.test import SimpleTestCase
+from ...templatetags.xforms_extras import \
```

Comment:
super nit, but i always like imports to be 

```
from foo import (
   bar,
   biz,
   baz,
)
```

so that diffs as imports are added or removed are easier to read